docs/src/further_topics/ugrid/operations.rst

```diff
@@ -469,145 +469,67 @@ SEM microscopy), so there is a wealth of tooling available, which
 :ref:`ugrid geovista` harnesses for cartographic plotting.
 
 GeoVista's default behaviour is to convert lat-lon information into full XYZ
-coordinates so the data is visualised on the surface of a 3D globe. The plots
-are interactive by default, so it's easy to explore the data in detail.
+coordinates so the data is visualised on the surface of a 3D globe; 2D
+projections are also supported. The plots are interactive by default, so it's
+easy to explore the data in detail.
 
-2D projections have also been demonstrated in proofs of concept, and will
-be added to API in the near future.
+Performing GeoVista operations on your :class:`~iris.cube.Cube` is made
+easy via this convenience:
+:func:`iris.experimental.geovista.cube_faces_to_polydata`.
 
-This first example uses GeoVista to plot the ``face_cube`` that we created
-earlier:
+Below is an example of using GeoVista to plot a low-res
+sample :attr:`~iris.cube.Cube.mesh` based :class:`~iris.cube.Cube`. For
+some truly spectacular visualisations of high-res data please see the
+GeoVista :external+geovista:doc:`generated/gallery/index`.
```

Comment:
[Rendered changes](https://trexfeathers-iris.readthedocs.io/en/additions_5740/further_topics/ugrid/operations.html#plotting)

I have:

- Removed the example of plotting a very simple mesh
- Changed the global example to use a mesh that is more readily available to the docs builder and to users.

I originally hoped to make this a running example, rather than a static one, which necessitated the big changes. However @bjlittle tells me that is non-trivial where PyVista is concerned. I have still kept most of those changes since it is now easier for a user to replicate, and will be easier for us to iterate in future.